File: Competicoes Diversas/2020/Rinha de Calouro/Pacotes/M/solver/solver.py
from math import ceil

S, D = [int(i) for i in input().split()]
cont = [0 for i in range(S)]
for d in range(D):
    t = [int(i) for i in input().split()]
    for i in range(S):
        cont[i] += t[i]

media = ceil(sum(cont) / D)
pref = cont.index(max(cont))

print(str(media))
print(str(pref + 1))

File: Competicoes Diversas/2020/Rinha de Calouro/Pacotes/M/solver/solver.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int S,D; cin >> S >> D;
    vector<long long> cont;
    cont.assign(S+1,0);
    long long sum = 0;
    for(int i=0; i<D; i++){
        for(int j=0; j<S; j++){
            long long x; cin >> x;
            cont[j] += x;
            sum += x;
        }
    }
    cout << ceil((double)sum/D) << endl;
    cout << 1 + distance(cont.begin(), max_element(cont.begin(), cont.end())) << endl;
}